internal/config/config.go

```diff
@@ -36,7 +36,7 @@ func InitConfig() {
 		viper.SetConfigName(".qbt")
 		viper.AddConfigPath(".") // optionally look for config in the working directory
 		viper.AddConfigPath(home)
-		viper.AddConfigPath("$HOME/.config/qbt") // call multiple times to add many search paths
+		viper.AddConfigPath(home + "/.config/qbt") // call multiple times to add many search paths
```

Comment:
Add this as an additional line and keep the old one so it doesn't break linux usage.

Reply:
Done.